### ISSUE TITLE
Not Sharing With PacketStream an d Repocket

### ISSUE THREAD

**Issue by dorel14** [MRColorR/money4band#33]
**Describe the bug**
Hello 
I have a problem with RePocket and PacketStream,  when i use them with money4Band , they don"'t share anything 
So i have tryed to install them with teir own docker installation and then i can see bandwith sharing 

[...]

**Device (please complete the following information):**
 - Type: Raspberry Pi 3 B+
 - OS details:  Linux Raspbian
 - Platform: arm64 (64bit)
 - Version: Raspbian Bullseyes 
 
 **Docker**
 - Docker Compose version: Docker Compose version v2.16.0
 - Docker version:  Version:           23.0.1
**To Reproduce**
1. Goto https://packetstream.io/dashboard/download and follow the instructions to install or go to https://hub.docker.com/r/repocket/repocket and follow instructions 

**Expected behavior**
Normally i should see bandwith sharing on Packetstream webpage 

**Screenshots**

**Additional context**

**Comment by MRColorR** [comment]
hi @dorel14  and thank for your message and glad you found a workaround :) 

Anyway, Could you share the logs from the two docker containers? So i can check what's the issue.
- For repocket could it be related to the containers errors they're discussing in their repocket Discord server (docker channel)? As they say a simple start and stop should make it work again ( or just reinstall it as you did). They need to find a better solution anyway and i think they're working on it, i'll look into it too.
- For packetstream i have no additional information on this issue, i'd like to dig in a log. i'll check my running machines asap and if you have more informations please let me know, thank you.

I'll release a new version of money4band after the completion of my master's degree project, atm the development is slowed down as i'm very busy.

**Comment by MRColorR** [comment]
Perfect, tank you.

For repocket I can confirm is a repocket software as a simple sudo docker restart repocket fixed the problem ( repocket logs showed a recurring stop message) the repocket devs are already aware of this bug and they are working on it.

My packetsream log seems to be fine so idk why is seems stuck. I'll try to restart also the the packetstream container maybe it should suffice.

I'll let you know if I find something

**Comment by dorel14** [comment]
For repocket , do you launch it with the api Key ? i ve seen that on their docker container repo 
![image](https://user-images.githubusercontent.com/10910133/222211130-37990d50-f634-48cd-b67f-d072b9f050b7.png)

I've tested to replace my password with my api key and it doesn't work more

**Comment by MRColorR** [comment]
it seems repocket devs changed the flag to be passed to the container, i'll rewrite the scripts and the docker compose file to adapt to the new flag but surely the don't care about retrocompatibility, not only for money4band but for all the users using their container.

thank you for letting me know @dorel14 i'll publish a new release soon

**Comment by MRColorR** [comment]
I just released a new version with a fix for this issue as well as adding an earnapp UUID import functionality and some minor improvements. 

I advise everyone to stop the old stack, download the new version and delete the old release, run the setup again and start the new stack.

I've already tested the new release on my raspberry Pi 4 and all the apps work fine. I'm closing the issue for now hoping this will fix it for good and the apps keep the current structure.
